Cargo.toml

```diff
@@ -0,0 +1,2 @@
+[workspace]
+members = ["webrender", "webrender_traits", "sample", "replay"]
```

Comment:
Total nit: typically, we would put each of these on their own line and sort them. e.g.:
```
[workspace]
members = [
  "replay",
  "sample",
  "webrender",
  "webrender_traits",
]
```